### ISSUE TITLE
Fails with OpenAPI::Modern 0.051

### ISSUE THREAD

**Issue by andk** [karenetheridge/Mojolicious-Plugin-OpenAPI-Modern#27]
Sample fail report: http://www.cpantesters.org/cpan/report/d866793c-8824-11ee-927a-aa7557dbe644

Statistical analysis hints that this started with 0.051:

``` text
****************************************************************
Regression 'mod:OpenAPI::Modern'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           1.0000          0.0000    9479727492773150.00
[1='eq_0.041']       -0.0000          0.0000      -1.56
[2='eq_0.042']       -0.0000          0.0000      -1.65
[3='eq_0.045']       -0.0000          0.0000      -2.04
[4='eq_0.046']       -0.0000          0.0000      -1.98
[5='eq_0.047']       -0.0000          0.0000      -0.86
[6='eq_0.048']       -0.0000          0.0000      -2.15
[7='eq_0.049']       -0.0000          0.0000      -2.60
[8='eq_0.050']       -0.0000          0.0000      -2.45
[9='eq_0.051']       -1.0000          0.0000    -7740165085975838.00
                 
R^2= 1.000, N= 51, K= 10
****************************************************************
```

**Comment by karenetheridge** [comment]
Looks like the new validation checks on `Content-Length` in the response is tripping up in the plugin, because I am validating the response before `fix_headers` is called on it (which is where Content-Length is set). I'll need to reorder some of the internals to work around this.

**Comment by karenetheridge** [comment]
fixed in 0.006.
